source/ota.c

```diff
@@ -249,14 +249,14 @@ static DocParseErr_t verifyRequiredParamsExtracted( const JsonDocParam_t * pMode
 static OtaErr_t validateUpdateVersion( const OtaFileContext_t * pFileContext );
 
 /**
- * @brief Check if the JSON can be parsed through a custom callback if initial parsing fails.
+ * @brief Check if the JSON can be parsed through the app callback if initial parsing fails.
  *
  * @param[in] pJson JSON job document.
  * @param[in] messageLength Length of the job document.
  * @return OtaJobParseErr_t OtaJobParseErrNone if successful, other error codes if failure.
  */
-static OtaJobParseErr_t parseJobDocFromCustomCallback( const char * pJson,
-                                                       uint32_t messageLength );
+static OtaJobParseErr_t handleCustomJob( const char * pJson,
+                                         uint32_t messageLength );
```

Comment:
messageLength should be size_t

Reply:
The `messageLength` parameter comes from [`parseJobDoc`](https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/main/source/ota.c#L293-L295) function. Changing the type across the function call chain would be out of scope of this PR.